src/Microsoft.Fast.Components.FluentUI/Components/DateTime/FluentCalendar.razor.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Microsoft.AspNetCore.Components;
+using Microsoft.Fast.Components.FluentUI.Utilities;
+
+namespace Microsoft.Fast.Components.FluentUI;
+
+/// <summary>
+/// Fluent Calendar based on
+/// https://github.com/microsoft/fluentui/blob/master/packages/web-components/src/calendar/.
+/// </summary>
+public partial class FluentCalendar : FluentCalendarBase
+{
+    public static string ArrowUp = "<svg width=\"24\" height=\"24\" viewBox=\"0 0 24 24\" fill=\"var(--neutral-fill-strong-focus)\" xmlns=\"http://www.w3.org/2000/svg\"><path d=\"M4.2 10.73a.75.75 0 001.1 1.04l5.95-6.25v14.73a.75.75 0 001.5 0V5.52l5.95 6.25a.75.75 0 001.1-1.04l-7.08-7.42a1 1 0 00-1.44 0L4.2 10.73z\"/></svg>";
```

Comment:
Can/should we use available icons here?

Reply:
Yes. When the "Icons PR" will be merge, I will update this section.
In the meantime, I allow the developer to modify this (static) value if he wishes to change the icon.